en/How to/Embedding web pages.md

```diff
@@ -0,0 +1,46 @@
+Learn how to use the [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) HTML element to embed web pages in your notes. 
```

Comment:
Do we want to somehow mention how to embed a web page in a canvas, which is much easier?